### PR TITLE
feat(history): Add ability to remove videos from watch history (#706)

### PR DESCRIPTION
**Pull request by dnicolson**
This allows removing a video from watch history using the `/feedback` endpoint.

Types haven’t been added as this endpoint seems unique unlike `IParsedResponse` types.

### CODE REVIEW

File: src/parser/youtube/History.ts [hunk 1]
```diff
@@ -25,4 +26,36 @@ export default class History extends Feed<IBrowseResponse> {
...
+    let feedbackToken;
+
+    for (const section of this.sections) {
+      for (const content of section.contents) {
+        const video = content as Video;
+        if (video.id === video_id) {
+          if (video.menu) {
+            feedbackToken = video.menu.top_level_buttons[0].endpoint.payload.feedbackToken;
+          }
+        }
```
Comment (rogerpadilla) [inline src/parser/youtube/History.ts:43]: This is missing a `break` so loops can be interrupted for better performance and clarity.

Additionally, nested `if` can be simplified into a single one.

```suggestion
        if (video.id === video_id && video.menu) {
            feedbackToken = video.menu.top_level_buttons[0].endpoint.payload.feedbackToken;
            break;
        }
```

**Review by rogerpadilla — COMMENTED** [review]
Add a quick comment.

**Review by rogerpadilla — APPROVED** [review]
LGTM!

**Review by LuanRT — APPROVED** [review]

**Comment by LuanRT** [comment]
Thank you!

**Pull request merged**